arch/arm/core/cortex_m/mpu/arm_mpu_v7_internal.h

```diff
@@ -105,6 +107,24 @@ static inline void _get_mpu_ram_region_attr(arm_mpu_region_attr_t *p_attr,
 
 	p_attr->rasr = _get_region_attr(1, ap, 1, 1, 1, 0, 0, size);
 }
+#endif /* USERSPACE || MPU_STACK_GUARD || APPLICATION_MEMORY */
+
+#ifdef CONFIG_NOCACHE_MEMORY
```

Comment:
minor: please use #if defined(CONFIG_NOCACHE_MEMORY) and #endif /* CONFIG_NO_CACHE_MEMORY */